script/Ciel/Day 13/13_01_02_CIEL13_6_1.txt

```diff
@@ -390,7 +391,7 @@ i hope youll praise me for this when you finally wake up
 [sha:8f996ac026da0476d79c9cea602b9af1c0f8b8bd]{
 -- Page 180, Offset 35789.
 -- やくそくをはたせなくてごめんなさい。
-I'm sorry for breaking our promise.
+im sorry for breaking our promise
```

Comment:
period can go but i think apostrophe should stay personally

Reply:
Same as above